examples/transfer_from_omnilock.rs

```diff
@@ -0,0 +1,375 @@
+
+/*
+# examples for the developer local node
+########################### sighash omnilock example #################################
+# 1. build a omnilock address
+ ./target/debug/examples/transfer_from_omnilock build \
+  --omnilock-tx-hash 34e39e16a285d951b587e88f74286cbdb09c27a5c7e86aa1b1c92058a3cbcc52 --omnilock-index 0  \
+  --receiver ckt1qyqt8xpk328d89zgl928nsgh3lelch33vvvq5u3024
+    # receiver lock-arg:b398368a8ed39448f95479c1178ff3fc5e316318
+    # {
+    #   "lock-arg": "0x00b398368a8ed39448f95479c1178ff3fc5e31631800",
+    #   "lock-hash": "0x6b845964aad7f568edf61a69d1c2278c68065dc91bad3c32234869aed86f7642",
+    #   "mainnet": "ckb1qqklkz85v4xt39ws5dd2hdv8xsy4jnpe3envjzvddqecxr0mgvrksqgqkwvrdz5w6w2y372508q30rlnl30rzcccqq2pnflw",
+    #   "testnet": "ckt1qqklkz85v4xt39ws5dd2hdv8xsy4jnpe3envjzvddqecxr0mgvrksqgqkwvrdz5w6w2y372508q30rlnl30rzcccqq3k897x"
+    # }
+# 2. transfer capacity to the address
+ckb-cli wallet transfer --from-account 0xc8328aabcd9b9e8e64fbc566c4385c3bdeb219d7 \
+  --to-address ckt1qqklkz85v4xt39ws5dd2hdv8xsy4jnpe3envjzvddqecxr0mgvrksqgqkwvrdz5w6w2y372508q30rlnl30rzcccqq3k897x \
+  --capacity 99 --tx-fee 0.001 --skip-check-to-address
+    # 0x999479f890a65cb4c37660565daeb77adec30cf65862e8e1aece09993b6340fc
+# 3. generate the transaction
+./target/debug/examples/transfer_from_omnilock gen --sender-key 8dadf1939b89919ca74b58fef41c0d4ec70cd6a7b093a0c8ca5b268f93b8181f \
+            --omnilock-tx-hash 34e39e16a285d951b587e88f74286cbdb09c27a5c7e86aa1b1c92058a3cbcc52 --omnilock-index 0  \
+            --receiver ckt1qyqy68e02pll7qd9m603pqkdr29vw396h6dq50reug \
+            --capacity 100.0 \
+            --tx-file tx.json
+# 4. sign the transaction
+./target/debug/examples/transfer_from_omnilock sign --sender-key 8dadf1939b89919ca74b58fef41c0d4ec70cd6a7b093a0c8ca5b268f93b8181f \
+            --omnilock-tx-hash 34e39e16a285d951b587e88f74286cbdb09c27a5c7e86aa1b1c92058a3cbcc52 --omnilock-index 0  \
+            --tx-file tx.json
+# 5. send transaction
+./target/debug/examples/transfer_from_omnilock send --tx-file tx.json
+*/
+
+#[derive(Args)]
+struct BuildOmniLockAddrArgs {
+    /// The receiver address
+    #[clap(long, value_name = "ADDRESS")]
+    receiver: Address,
+
+    /// omnilock script deploy transaction hash
+    #[clap(long, value_name = "omnilock_tx_hash")]
+    omnilock_tx_hash: H256,
+
+    /// cell index of omnilock script deploy transaction's outputs
+    #[clap(long, value_name = "index")]
+    omnilock_index: usize,
+
+    /// CKB rpc url
+    #[clap(long, value_name = "URL", default_value = "http://127.0.0.1:8114")]
+    ckb_rpc: String,
+}
+#[derive(Args)]
+struct GenTxArgs {
+    /// The sender private key (hex string)
+    #[clap(long, value_name = "KEY")]
+    sender_key: H256,
+    /// The receiver address
+    #[clap(long, value_name = "ADDRESS")]
+    receiver: Address,
+
+    /// omnilock script deploy transaction hash
+    #[clap(long, value_name = "omnilock_tx_hash")]
```

Comment:
```suggestion
    #[clap(long, value_name = "H256")]
```

The `value_name` here is actually the argument type for cli help message.